### PR TITLE
Update servers_v7.json

### PR DESCRIPTION
**Pull request by maxutka99**
Add devass.su server

### CODE REVIEW

**Comment by Darkness6030** [comment]
You should add it at the bottom of list

**Comment by Darkness6030** [comment]
Or the pull won't be merged

**Comment by maxutka99** [comment]
@Darkness6030 please merge

**Comment by Darkness6030** [comment]
Anuke will merge :-/ 
I am just a random guy who saw ur PR

File: servers_v7.json [hunk 1]
```diff
@@ -50,5 +50,8 @@
   {
     "name": "Shiza Minigames",
     "address": ["shizashizashiza.ml"]
+  },
+  { "name": "devass.su",
```
Comment (Anuken) [inline servers_v7.json:54]: incorrect formatting, name must be on new line like all the other servers

**Comment by maxutka99** [comment]
@Anuken please merge

**Comment by maxutka99** [comment]
thanks :)

**Pull request merged**